packages/core/src/test/shared/utilities/pathFind.test.ts

```diff
@@ -42,4 +41,93 @@ describe('pathFind', function () {
         const regex = /bin[\\\/](code|code-insiders)$/
         assert.ok(regex.test(vscPath), `expected regex ${regex} to match: "${vscPath}"`)
     })
+
+    describe('findSshPath', function () {
+        let previousEnv: NodeJS.ProcessEnv
+
+        beforeEach(function () {
+            previousEnv = testutil.readEnv()
+        })
+
+        afterEach(function () {
+            testutil.setEnv(previousEnv)
```

Comment:
if `readEnv` returns a shallow copy, then this becomes much more conventional/familiar:
```suggestion
            process.env = previousEnv
```